dftimewolf/lib/exporters/local_filesystem.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+"""Export processing results to Timesketch."""
+
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import shutil
+
+from dftimewolf.lib.exporters.exporters import BaseExporter
+
+
+class LocalFilesystemExporter(BaseExporter):
+  """Export results of a collector or processor to the local filesystem.
+
+  Attributes:
+    previous_output: (name, path) tuple to export.
+    directory: directory into which files should be copied.
```

Comment:
Please add type information - is directory a string? Or an os.path? Relative or absolute? Trailing slash?